### PR TITLE
Adds a Beefman as an AI Core display option

### PR DESCRIPTION
**Pull request by franklinfurter**
:)
## About The Pull Request

Allows AI players to select a Beefman as their AI Core display. Has a cool death animation and everything.

**Normal Animation** 
franklin-core-background

**Death Transition Animation**
franklin-core-death-trans-loop

**Death Animation**
franklin-core-death-loop

## Why It's Good For The Game

Considering Beefmen are out of the game for the time being I felt as if this would be fitting to add in.

## Changelog
:cl:
add: New Beefman AI core display
/:cl:

### CODE REVIEW

**Comment by manray0** [comment]
looks dumb.

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -165,6 +166,9 @@ GLOBAL_LIST_INIT(uplink_spawn_loc_list, list(UPLINK_PDA, UPLINK_RADIO, UPLINK_PE
 	//Female Uniforms
 GLOBAL_LIST_EMPTY(female_clothing_icons)
 
+	//radical shit
+GLOBAL_LIST_INIT(hit_appends, list("-OOF", "-ACK", "-UGH", "-HRNK", "-HURGH", "-GLORF"))
```
Comment (EgorDinamit) [inline code/_globalvars/lists/flavor_misc.dm]: What the fuck is it.

Comment (franklinfurter) [reply code/_globalvars/lists/flavor_misc.dm]: I don't know I didn't touch that.

Comment (EgorDinamit) [reply code/_globalvars/lists/flavor_misc.dm]: Then you should remove it.

Comment (franklinfurter) [reply code/_globalvars/lists/flavor_misc.dm]: I dont know what it does.

Comment (SgtHunk) [reply code/_globalvars/lists/flavor_misc.dm]: Franklin moment

Comment (franklinfurter) [reply code/_globalvars/lists/flavor_misc.dm]: deleting it

**Comment by EgorDinamit** [comment]
Dude, you just editted **/tg/ dmi file.**

**Comment by SgtHunk** [comment]
Franklin you edited tg dmi
You are going to lose subscriber

**Comment by franklinfurter** [comment]
oh my fucking god

**Comment by franklinfurter** [comment]
files edited only shows the thing i added what are you talking about

**Comment by BarrelOfNapkins** [comment]
Fronk is new to coding; if you have criticism I recommend telling him what to do along with what he shouldn't do

**Comment by franklinfurter** [comment]
yes please what Barrel said

**Comment by JohnFulpWillard** [comment]
FRANKLIN STOP IT FROM GOING STALE

**Comment by franklinfurter** [comment]
HOW I DO THAT

**Comment by remove32** [comment]
Hahahaha

**Comment by MaltVinegar** [comment]
IT'S STALE FRANKLIN

**Comment by franklinfurter** [comment]
HELP HOW DO I MAKE IT NO T STALE HELP

**Comment by franklinfurter** [comment]
WIND STOP LAUGHING AT ME
 STOP

**Comment by E231kei500bandai** [comment]
you need to make change to make it not stale i think oki

**Comment by E231kei500bandai** [comment]
idk i dont code

**Comment by franklinfurter** [comment]
AHHHHHHHHHHHHHHHH

**Pull request closed without merging**